src/olympia/promoted/admin.py

```diff
@@ -32,13 +33,15 @@ class PromotedSubscriptionInline(admin.StackedInline):
         'checkout_cancelled_at',
         'checkout_completed_at',
         'cancelled_at',
+        'stripe_information',
```

Comment:
I'm thinking that we might have more info to display later, hence the generic custom field